homeassistant/components/discovery.py

```diff
@@ -43,8 +44,16 @@
     'apple_tv': ('media_player', 'apple_tv'),
 }
 
+CONF_IGNORE = 'ignore'
+
+# Validates that all entries are part of SERVICE_HANDLERS
+IGNORE_SCHEMA_LIST = vol.All(cv.ensure_list,
+                             [vol.Any(*list(SERVICE_HANDLERS.keys()))])
```

Comment:
What?! Don't you like magic? 🤣 I missed that one, ``vol.In`` is exactly what I'm after!